docs/real-time-analytics/event-streams/add-manage-eventstream-destinations.md

```diff
@@ -29,7 +29,7 @@ The following destinations are supported by Fabric Eventstream:
 
 | Destinations          | Description |
 | --------------- | ---------- |
-| Custom app | With this destination, you can easily route your real-time events to a custom application. It allows you to connect your own applications to the eventstream and consume the event data in real time. It's useful when you want to ingest data from external system into Microsoft Fabric.  |
+| Custom app | With this destination, you can easily route your real-time events to a custom application. It allows you to connect your own applications to the eventstream and consume the event data in real-time. It's useful when you want to egress real-time data to an external system living outside Microsoft Fabric.  |
```

Comment:
```suggestion
| Custom app | With this destination, you can easily route your real-time events to a custom application. It allows you to connect your own applications to the eventstream and consume the event data in real time. It's useful when you want to egress real-time data to an external system living outside Microsoft Fabric.  |
```
"Real time" is not hyphenated as a noun but is hyphenated as an adjective, per the Microsoft style guide.